lib/govuk_ab_testing/minitest_helpers.rb

```diff
@@ -0,0 +1,29 @@
+module GovukAbTesting
+  module MinitestHelpers
+    def with_variant(args)
+      ab_test_name, variant = args.first
+
+      ab_test = GovukAbTesting::AbTest.new(ab_test_name)
+
+      previous_variant = @request.headers[ab_test.request_header]
+      @request.headers[ab_test.request_header] = variant
+
+      yield
+
+      @request.headers[ab_test.request_header] = previous_variant
+    end
+
+    def assert_ab_test_rendered(ab_test_name)
```

Comment:
I'm not sure "rendered" is quite the right wording. Maybe more like `assert_ab_test_correctly_set_up` ? Also, is there any way to run this automatically if they've used `with_variant` ? Otherwise you have to explicitly remember to call this test helper, which probably means you've remembered to set up the header/meta tag anyway.